tests/ui/suggestions/nested-non-tuple-tuple-struct.stderr

```diff
@@ -9,8 +9,8 @@ LL |     let _x = (S { x: 1.0, y: 2.0 }, S { x: 3.0, y: 4.0 });
    |
 help: `S` is a tuple struct, use the appropriate syntax
    |
-LL |     let _x = (S(/* fields */), S { x: 3.0, y: 4.0 });
-   |               ~~~~~~~~~~~~~~~
+LL |     let _x = (S(/* f32 */, /* f32 */), S { x: 3.0, y: 4.0 });
+   |               ~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@nnethercote this is a case where the "reuse the literal values that were already there" gets harder: should it have been `S(3.0, 4.0)` or `S(4.0, 3.0)`?

Reply:
I'm guessing these help suggestions aren't required to be 100% correct, right? Though of course we want them to be as close to 100% as reasonable.

So I would answer `S(3.0, 4.0)`, because that matches the order the float literals appear in the code. Sure, it's possible that somebody wrote the literal with the fields in a different order than in the type declaration, but I would expect that is less likely than matching the type declaration order.

Reply:
I think if we ever handle this case, I'd only hazard a guess on moving existing code when there is no ambiguity, and leave the comment placeholder for cases like this one so that the user *has* to make a determination manually (with IDEs a user could accidentally accept a suggestion without noticing the subtlety of what happened).